src/infrastructure/database/migrations/versions/2024_06_21_11_24-flow_assessments.py

```diff
@@ -0,0 +1,68 @@
+"""Flow assessments
+
+Revision ID: 297c9d675e2f
+Revises: 62843fdc3466
+Create Date: 2024-05-15 11:24:20.074328
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "297c9d675e2f"
```

Comment:
Check revision ids please, in both migrations, revision id seems same.

Reply:
Its looks like good for me maybe i miss something? )
```alembic history                         
c587d336f28e -> 297c9d675e2f (head), Flow assessments
01115b529336 -> c587d336f28e, Library default keywords
edb2781f141b -> 01115b529336, Link subjects and answers
62843fdc3466 -> edb2781f141b, Add subjects
23fe6ccfb031 -> 62843fdc3466, create encryption internal function
736adb0ea547 -> 23fe6ccfb031, Add timezone offset to answers
54123357967a -> 736adb0ea547, Create function to decrypt on database level
46f285831ae8 -> 54123357967a, transfer_from_to_user_id_columns
47405ad1f948 -> 46f285831ae8, Add stream ip address and port to applet
3fb536a58c94 -> 47405ad1f948, remove duplicated pending invitations```

Reply:
Sorry, my bad, didn't notice one migration was for arbitrary database